wolfcrypt/src/port/iotsafe/iotsafe.c

```diff
@@ -498,8 +529,10 @@ static int iotsafe_getrandom(unsigned char* output, unsigned long sz)
         else
             ret = 0;
     }
-    if (expect_tok(NULL, 0, NULL, NULL) < 0) {
-        ret = WC_HW_E;
+    for (i = 0; i < 3; i++) {
```

Comment:
Why the for loop here trying 3 times? Can you expand with comment? Should the 3 be an overridable macro?